spec/requests/api/v1/record/merchants_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+describe 'Merchants record endpoints' do 
+    it 'can see a list of merchants' do
+        create_list(:merchant, 5)
+
+        get '/api/v1/merchants'
+
+        expect(response).to be_successful
```

Comment:
This `expect` may be helpful in initial testing, but it is redundant now. This test will fail if the response is not successful with the `expect` on line 12.

Tests are a form of coupling and can often make it harder to deal with changing business requirements. Don't sacrifice test coverage, just make sure your tests cover your functionality without making it too hard to change the code.